vendor/assets/javascripts/trumbowyg/vendor/plugins/resizimg/trumbowyg.resizimg.custom.js

```diff
@@ -56,6 +56,16 @@
                         return ev.preventDefault();
                     };
 
+                    var setImgMeta = function(img) {
+                      if (!img[0]) return;
```

Comment:
correct me when I'm wrong, but can't there be more  that 1 image in the editor?
so, shouldn't we iterate over all, not only one?

Reply:
Yeap, you are right. Will update the function